dbaas/dbaas-aggregator/src/main/java/com/netcracker/cloud/dbaas/exceptions/ForbiddenException.java

```diff
@@ -0,0 +1,10 @@
+package com.netcracker.cloud.dbaas.exceptions;
+
+import com.netcracker.cloud.core.error.runtime.ErrorCode;
+import com.netcracker.cloud.core.error.runtime.ErrorCodeException;
+
+public class ForbiddenException extends ErrorCodeException {
+    public ForbiddenException(ErrorCode errorCode, String detail) {
```

Comment:
Consider adding parameter validation to ensure errorCode and detail are not null.

```suggestion
    public ForbiddenException(ErrorCode errorCode, String detail) {
        super(Objects.requireNonNull(errorCode, "errorCode must not be null"), Objects.requireNonNull(detail, "detail must not be null"));
    }
```

#deepseek-review:inline